fw/rbcx-coprocessor/src/MotorController.cpp

```diff
@@ -117,28 +142,32 @@ void motorDispatch(const CoprocReq_MotorReq& request) {
         }
         targetMotor.setTargetVelocity(ticksPerSec);
     } break;
+    case CoprocReq_MotorReq_homePosition_tag:
+        targetMotor.homePosition(request.motorCmd.homePosition);
+        break;
     case CoprocReq_MotorReq_setPosition_tag:
-        targetMotor.setTargetPosition(request.motorCmd.setPosition);
+        targetMotor.setTargetPosition(request.motorCmd.setPosition, false);
         break;
     case CoprocReq_MotorReq_addPosition_tag:
-        targetMotor.addTargetPosition(request.motorCmd.addPosition);
+        targetMotor.setTargetPosition(request.motorCmd.addPosition, true);
+        break;
+    case CoprocReq_MotorReq_setVelocityRegCoefs_tag:
+        targetMotor.setVelocityPid(request.motorCmd.setVelocityRegCoefs);
+        break;
+    case CoprocReq_MotorReq_setPositionRegCoefs_tag:
+        targetMotor.setPositionPid(request.motorCmd.setPositionRegCoefs);
+        break;
+    case CoprocReq_MotorReq_setConfig_tag:
+        targetMotor.setConfig(request.motorCmd.setConfig);
         break;
-    case CoprocReq_MotorReq_setVelocityRegCoefs_tag: {
-        auto& coefs = request.motorCmd.setVelocityRegCoefs;
-        targetMotor.setVelocityPid(coefs.p, coefs.i, coefs.d);
-    } break;
-    case CoprocReq_MotorReq_setPositionRegCoefs_tag: {
-        auto& coefs = request.motorCmd.setPositionRegCoefs;
-        targetMotor.setPositionPid(coefs.p, coefs.i, coefs.d);
-    } break;
     }
 }
 
 void motorReset() {
     std::scoped_lock lock(motorMut);
 
     for (int idx : { 0, 1, 2, 3 }) {
-        motor[idx].setTargetPower(0);
+        motor[idx] = Motor();
```

Comment:
Tady je potřeba přenést m_lastEncTicks, jinak další `.poll()` pohne s `m_actualPosition` o tolik, kolik je momentálně na Enkodér periferii napočítáno.
Pokud zrovna bězí motory, pohne se sice dál, dokud se nedotočí, ale nebude to o tolik snad.